ibm/service/resourcecontroller/resource_ibm_resource_instance.go

```diff
@@ -869,10 +869,18 @@ func waitForResourceInstanceCreate(d *schema.ResourceData, meta interface{}) (in
 				}
 				return nil, "", fmt.Errorf("[ERROR] Get the resource instance %s failed with resp code: %s, err: %v", d.Id(), resp, err)
 			}
-			if *instance.State == RsInstanceFailStatus {
-				return instance, *instance.State, fmt.Errorf("[ERROR] The resource instance '%s' creation failed: %v", d.Id(), err)
+			if instance.LastOperation != nil && instance.LastOperation.Async != nil && *instance.LastOperation.Async {
+				if *instance.LastOperation.State == RsInstanceFailStatus {
+					return instance, *instance.LastOperation.State, fmt.Errorf("[ERROR] The resource instance '%s' create failed during async operation. error: %v last operation description: %+v", d.Id(), err, *instance.LastOperation.Description)
```

Comment:
Can we add a newline for `last operation` `error: %v\n last operation description: %+v`?
Also, can we say `last_operation_description` instead of `last operation description`